config/initializers/cors.rb

```diff
@@ -7,7 +7,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins '*'
+    origins 'localhost:3000', 'localhost:4000', 'https://one-minute-writer.herokuapp.com'
```

Comment:
Do we need an origin for the Python heroku?

Reply:
Don't think so, Python might need ours however...